src/cpp/fastdds/core/policy/ParameterSerializer.hpp

```diff
@@ -629,9 +629,12 @@ inline bool ParameterSerializer<ParameterPropertyList_t>::read_content_from_cdr_
     parameter.length = parameter_length;
 
     uint32_t pos_ref = cdr_message->pos;
-    uint32_t num_properties;
+    uint32_t num_properties = 0;
     bool valid = fastrtps::rtps::CDRMessage::readUInt32(cdr_message, &num_properties);
-
+    if (!valid)
+    {
+      return false;
```

Comment:
I know we don't have it publicly stated, but we use 4 spaces for indentation.
```suggestion
        return false;
```